node/test/unit/tessel.js

```diff
@@ -565,7 +565,20 @@ exports['Tessel.Port.prototype'] = {
     test.done();
   },
 
-  I2C: function(test) {
+  I2CnoArgsAlwaysHasPort: function(test) {
+    test.expect(3);
+
+    var device1 = new this.port.I2C();
```

Comment:
Also, that is tested here: https://github.com/tessel/t2-firmware/pull/170/files/2e4ad5c6c15ec91d282155e9c9a739cb595724ad#diff-4df274bad3cd8322e337b8eac6af233bR1644